bigquery/system-test/list_datasets_and_projects_test.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2016, Google, Inc.
```

Comment:
Can you also write unit tests? Here's an example: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/storage/test/buckets.test.js